Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -291,24 +291,23 @@ def return_metrics():
         return_results(API_POINTS_TABLE)
 
 
-def rerun_command_if_required(api_res: dict, rate_limit_auto_retry: bool):
+def rerun_command_if_required(api_res: dict, retry_on_rate_limit: bool):
     daily_points_remaining = dict_safe_get(api_res, ('bucket_info', 'daily_points_remaining'), 0)
-    if not (rate_limit_auto_retry and daily_points_remaining):
-        results = CommandResults(
-            readable_output=(
-                f'Error in API call to AutoFocus:\nMessage: {api_res.get("message")}\n'),
-            entry_type=EntryType.ERROR
-        )
-    else:
-        next_run_in_seconds = int(dict_safe_get(api_res, ('bucket_info', 'wait_in_seconds'), 70)) + 1  # type: ignore
+    if retry_on_rate_limit and daily_points_remaining:
+        next_run_in_seconds = min(int(dict_safe_get(api_res, ('bucket_info', 'wait_in_seconds'), 70)) + 20, 300)  # type: ignore
```

Comment:
if wait_in_seconds from the api is 2 hours for some reason, we should just not retry at all. In this case we will be polling every five minutes for no reason.
should have a 
if next_run_in_seconds > 10 minutes as part of this if above